core/src/main/scala/io/chrisdavenport/rediculous/RedisCommands.scala

```diff
@@ -696,11 +696,15 @@ object RedisCommands {
   def expire[F[_]: RedisCtx](key: String, seconds: Long): F[Boolean] = 
     RedisCtx[F].keyed(key, NEL.of("EXPIRE", key.encode, seconds.encode))
 
-  def mget[F[_]: RedisCtx](key: String): F[List[Option[String]]] = {
+  private[this] def mget[F[_]: RedisCtx](key: String): F[List[Option[String]]] = {
     val cmd = NEL("MGET", key.encode :: Nil)
     RedisCtx[F].keyed(key, cmd)
   }
 
+  def mget[F[_]: RedisCtx](key: String, keys: String*): F[List[Option[String]]] = {
```

Comment:
@ChristopherDavenport I told him that I thought this `(head, tail*)` pattern was necessary to disambiguate the overloads. Let me know if I'm wrong there.